lib/tabs/KTabsList.vue

```diff
@@ -63,7 +65,22 @@
         </slot>
       </button>
     </template>
-  </div>
+    <template #more="{ overflowItems }">
+      <KIconButton
+        tooltip="More"
+        icon="optionsHorizontal"
+        :style="overflowButtonStyles()"
```

Comment:
An improvement we can discuss is how can we improve the visibility of the selected tab when it is overflowed.